packages/desktop/package.json

```
@@ -1,7 +1,7 @@
 {
     "name": "desktop",
-    "productName": "Firefly",
-    "version": "1.2.0",
+    "productName": "Firefly Beta",
+    "version": "1.3.0-beta",
```

Change before merge:
```suggestion
    "productName": "Firefly",
    "version": "1.3.0",
```